README.md

```diff
@@ -312,3 +312,18 @@ Another possibility is that the device wasn't provisioned with the certificate
 that's on NervesHub.
 
 See also [NervesHubWeb: Potential SSL Issues](https://github.com/nerves-hub/nerves_hub_web#potential-ssl-issues)
+
+## Internal API Versions
+
+`NervesHubLink.Configurator` includes two internal versions for NervesHub to determine what features are available on the device socket.
+
+### `device_api_version`
+
+- `1.0.0` - Updating firmware, status updates, reboot device
+- `2.0.0` - Identify a device, archives
+- `2.1.0` - Run scripts on a device separate from the console, health checks, syncing firmware and archive keys
```

Comment:
Health Checks are still in PR unreleased.

Also, what does syncing firmware mean?

Reply:
It's syncing firmware keys and archive keys, I'll update it so it's more obvious